### PR TITLE
Fixed issues caused by changes to HashMap in godot

### PR DESCRIPTION
**Pull request by Trey2k**
Just a quick fix for changes to hashmaps in the master godot branch https://github.com/godotengine/godot/commit/8b7c7f5a753b43cec10f72b274bb1d70c253652b

### CODE REVIEW

File: terrain/fixed_lod/voxel_terrain.cpp [hunk 1]
```diff
@@ -377,8 +377,9 @@ void VoxelTerrain::set_block_enter_notification_enabled(bool enable) {
 
 	if (enable == false) {
 		const Vector3i *key = nullptr;
```
Comment (Zylann) [inline terrain/fixed_lod/voxel_terrain.cpp:379]: Key is now unused

File: terrain/fixed_lod/voxel_terrain.cpp [hunk 2]
```diff
@@ -377,8 +377,9 @@ void VoxelTerrain::set_block_enter_notification_enabled(bool enable) {
...
-		while ((key = _loading_blocks.next(key))) {
-			LoadingBlock *lb = _loading_blocks.getptr(*key);
+		
+		for (auto elm = _loading_blocks.begin(); elm < _loading_blocks.end(); ++elm) {
+			LoadingBlock *lb = _loading_blocks.getptr(elm->key);
```
Comment (Zylann) [inline terrain/fixed_lod/voxel_terrain.cpp:382]: Can you use the iterator instead of doing a second hashmap lookup?

**Review by Zylann — CHANGES_REQUESTED** [review]

File: terrain/fixed_lod/voxel_terrain.cpp (hunk 2 above)
Comment (Trey2k) [reply terrain/fixed_lod/voxel_terrain.cpp]: Just pushed changes. Sorry did even noticed I did a redundant lookup.

**Comment by Zylann** [comment]
Thanks!

**Pull request merged**